docs/visual-basic/misc/the-remote-server-machine-does-not-exist-or-is-unavailable.md

```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: The remote server machine does not exist or is unavailable (Visual Basic)"
```

Comment:
Why do some of these get (Visual Basic) appended to the description while others in the vb folder don't?

Reply:
The text is coming from the H1, not the metadata title.